metricbeat/module/php_fpm/_meta/config.yml

```diff
@@ -0,0 +1,6 @@
+- module: php_fpm
```

Comment:
Ah, haproxy has a config.yml and a config.full.yml, that is why it also works there.